src/EnergyPlus/IOFiles.cc

```diff
@@ -456,6 +456,9 @@ void IOFiles::OutputControl::getInput(EnergyPlusData &state)
             { // "sqlite"
                 sqlite = boolean_choice(find_input(fields, "output_sqlite"));
             }
+            { // "psz"
+                psz = boolean_choice(find_input(fields, "output_plant_component_sizing"));
+            }
```

Comment:
Triggers no csv output if user has OutputControl:Files input set to No.

    state.files.psz.ensure_open(state, "SizePlantLoop", state.files.outputControl.psz);